src/Debug.c

```diff
@@ -46,6 +47,36 @@ void Ros_Debug_BroadcastMsg(char* fmt, ...)
 
     if (ros_DebugSocket == -1)
         Ros_Debug_Init();
+    
+   // Timestamp
+
+    time_t DEBUG_MSG_TIMESTAMP;
+    struct tm ts;
+    struct timeval tv;
+
+
+    time(&DEBUG_MSG_TIMESTAMP);
+
+    localtime_r(&DEBUG_MSG_TIMESTAMP, &ts);
+
+    strftime(Formatted_Time, sizeof(Formatted_Time), "%a %Y-%m-%d %H:%M:%S", &ts);
+
+    gettimeofday(&tv, NULL);
```

Comment:
Just wondering: wouldn't this return a *different* time point (compared to the `DEBUG_MSG_TIMESTAMP`)?

Reply:
I just realized it now that you said that. Thanks for pointing that out. Will change it out and use the host time synced using the micro-ROS Agent instead.